### ISSUE TITLE
Openshift Console

### ISSUE THREAD

**Issue by saashqdev** [OpenUnison/openunison-k8s#136]
Hi Marc:

Has anyone tried to put the Openshift Console on Openunison?

Cheers,
Dave

**Comment by mlbiam** [comment]
Yes.  It's been a while but OpenUnison works well as an OIDC provider for OpenShift.  For SSO, you can combine the instructions from OpenUnison (https://openunison.github.io/documentation/custom-sso/#extending-the-kubernetes-trust) and https://docs.openshift.com/container-platform/4.17/authentication/identity_providers/configuring-oidc-identity-provider.html.  Based on the docs OpenShift will just-in-time provision your oidc claims into their custom object, so i don't think you need to worry about OpenUnison just-in-time provisioning that anymore.

**Comment by saashqdev** [comment]
That's good to know, thanks.
